src/azure-cli-core/azure/cli/core/tests/test_profile.py

```diff
@@ -536,6 +536,7 @@ def test_login_in_cloud_shell(self, cloud_shell_credential_mock, create_subscrip
 
     @mock.patch('requests.get', autospec=True)
     @mock.patch('azure.cli.core._profile.SubscriptionFinder._create_subscription_client', autospec=True)
+    @mock.patch.dict('os.environ', {'AZURE_CORE_USE_MSAL_MANAGED_IDENTITY': 'false'})
     def test_login_with_mi_system_assigned(self, create_subscription_client_mock, mock_get):
```

Comment:
Is there any plan to drop `msrestazure`?

Also, since we have `_msal` for tests using msal, is it worth adding the suffix `_msrestazure` to tests that use msrestazure?